runtime/doc/lua.txt

```diff
@@ -2970,6 +2979,30 @@ vim.fs.normalize({path}, {opts})                          *vim.fs.normalize()*
 
         vim.fs.normalize('$XDG_CONFIG_HOME/nvim/init.vim')
         -- '/Users/jdoe/.config/nvim/init.vim'
+
+        vim.fs.normalize('~/src/nvim/api/../tui/./tui.c')
+        -- '/home/jdoe/src/nvim/tui/tui.c'
+
+        vim.fs.normalize('.')
+        -- '.'
+
+        vim.fs.normalize('./foo/bar')
+        -- 'foo/bar'
+
+        vim.fs.normalize('foo/../../../bar')
+        -- '../../bar'
+
+        vim.fs.normalize('/home/jdoe/../../../bar')
+        -- /bar
+
+        vim.fs.normalize('C:foo/../../baz')
+        -- 'C:../baz'
+
+        vim.fs.normalize('C:/foo/../../baz')
+        -- C:/baz
```

Comment:
this is a very noisy way to show input/output examples.

usage examples are intended to help with the mechanics of using an interface. for input/output we can just have a table that shows input/output

```
"./foo/bar"               => "foo/bar"
"foo/../../../bar"        => "../../bar"
"/home/jdoe/../../../bar" => "/bar"
"C:foo/../../baz"         => "C:../baz"
"C:/foo/../../baz"        => "C:/baz"
```

Reply:
I'm not sure how to make the `=>`s align in the Vimdocs, I aligned them in the Lua docs but the vimdocs generator removes the extra spaces

Reply:
use a codeblock